app/(candidates)/candidates/candidate-dashboard/my-page/ContentBlocksEditor.tsx

```
@@ -0,0 +1,525 @@
+  const videoInputRef = useRef<HTMLInputElement | null>(null);
+
+  /* ----- Render each block type inline-editable ----- */
+  let inner: React.ReactNode;
+  switch (block.type) {
+    case "HEADING":
+      const headingClass =
+        block.level === 1
+          ? `text-4xl font-bold ${color} px-2 py-1`
+          : `text-2xl font-semibold ${color} px-2 py-1`;
+      inner = (
+        <h2
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)